### PR TITLE
fix: fix tooltip of tempchart

### LINKED ISSUE #1655: Temperature chart tooltip/legend disappears
### Mainsail Version:

v2.8.0

### Browser:

Chrome

### Device:

Desktop PC

### Operating System:

Windows

### What happened?

Temperature chart tooltip/legend disappears almost immediately after stopping the mouse movement making almost impossible to read the temperature values.

### What did you expect to happen instead?

Temperature chart tooltip/legend should be visible even after stopping the mouse movement.

### How to reproduce this bug?

- go to Temperatures chart
- move mouse, tooltip/legend is visible
- stop moving mouse, tooltip/legend should disappear within a second

### Additional information:

Confirmed the same issue in Edge
Checked fluidd, it's working fine there

### PR DESCRIPTION
**Pull request by meteyou**
## Description

This PR will change to setOption in the tempchart and will stop the chart update, until the mouse if hovered.

## Related Tickets & Documents

fixes #1655 

## Mobile & Desktop Screenshots/Recordings

none

## [optional] Are there any post-deployment tasks we need to perform?

none

### CODE REVIEW

File: src/components/charts/TempChart.vue [hunk 1]
```diff
@@ -273,11 +278,14 @@ export default class TempChart extends Mixins(BaseMixin, ThemeMixin) {
     tooltipFormatter(datasets: any) {
         let output = ''
 
+        //window.console.log('tempchartFix', datasets)
```
Comment (rackrick) [inline src/components/charts/TempChart.vue:281]: remove commented code

File: src/components/charts/TempChart.vue [hunk 2]
```diff
@@ -273,11 +278,14 @@ export default class TempChart extends Mixins(BaseMixin, ThemeMixin) {
...
+
         const mainDatasets = datasets.filter((dataset: any) => dataset.seriesName.endsWith('-temperature'))
         if (datasets.length) {
             let outputTime = datasets[0]['axisValueLabel']
+            //window.console.log('tempchartFix', outputTime)
```
Comment (rackrick) [inline src/components/charts/TempChart.vue:286]: remove commented code

File: src/components/charts/TempChart.vue [hunk 3]
```diff
@@ -273,11 +278,14 @@ export default class TempChart extends Mixins(BaseMixin, ThemeMixin) {
...
             outputTime = outputTime.substring(outputTime.indexOf(' '))
-            let theme = $vuetify.theme.dark ? 'theme-dark' : ''
+            const theme = this.$vuetify.theme.dark ? 'theme-dark' : ''
```
Comment (rackrick) [inline src/components/charts/TempChart.vue:288]: this was fixed in pr #1706 ?

Comment (meteyou) [reply src/components/charts/TempChart.vue:288]: yes, because i started with the 1706 branch with this fix. i will update this branch...

**Review by rackrick — APPROVED** [review]

**Pull request merged**